### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by darrenburns** [SethClydesdale/genki-study-resources#182] [General]
This is an incredible resource. It's helping me so much. Thank you for all of the effort you and other contributors have put into it!

(That's all I wanted to say :) feel free to close this discussion - I just wanted to get this message to you.)

**Comment by SethClydesdale** [comment]
You're very welcome! I'm happy to hear that it's been helping you! It motivates me to continue working on projects like these :)

日本語の勉強頑張ってね٩( 'ω' )و
